index.html

```diff
@@ -64,6 +64,36 @@
         </gu-column>
       </gu-row>
 
+      <h2>Boxes</h2>
+      <gu-row>
+        <gu-column span="4">
+          <gu-box variant="primary">
+            This is some content
+          </gu-box>
+        </gu-column>
+        <gu-column span="4" gu-box="secondary">
```

Comment:
I think in this case it's okay as using OOCSS I would happily write:

```html
<div class="col-3 box padded rounded">content</div>
```

which in this case (without padded rounded) would result in:

```html
<gu-column span="3" gu-box="primary">content<gu-column>
```

again, this is really for decoration. Essentially it just adds a class. The problem your referring to would still exist if we were just using raw css and this is why OOCSS exists, you write semantic classes which only apply small rule sets. You also then enforce them with compound selectors, which provides a kind of scoping. I guess the key is that, when we do use these attribute directives we limit the behaviour. I would suggest limiting it to only assigning classes. What do you think? 

PS: Personally, I think `display: flex` in particular is a special case as it is coupled to a parent/child relationship within the html whereas most css rules don't have this restriction. When they are coupled to the html its usually that only certain rules apply to certain element types.